.github/workflows/_integration_test.yml

```diff
@@ -47,7 +47,20 @@ jobs:
         id: 'auth'
         uses: google-github-actions/auth@v2
         with:
-          credentials_json: '${{ secrets.GOOGLE_CREDENTIALS }}'
+          credentials_json: '{
```

Comment:
guessing this was not meant to be checked in, probably want to void these credentials